packages/components/src/Product/HblPremium.purs

```diff
@@ -0,0 +1,21 @@
+module KSF.Product.HblPremium where
+
+import Prelude
+
+import Data.Array as Array
+import Data.Int (toNumber)
+import Data.Maybe (maybe)
+import KSF.Api.Package (Package)
+import KSF.Product as Product
+import React.Basic (JSX)
+
+hblPremium :: Product.Product -> JSX
+hblPremium p = Product.product p
+
+toProduct :: Package -> Product.Product
+toProduct  { name, id, offers } =
+  { name: name
+  , id:   id
+  , description: "Alla artiklar på hbl.fi"
+  , price: maybe 100.0 (_.monthlyPrice >>> toNumber >>> (_ / 100.0)) $ Array.head offers
```

Comment:
I don't get this, can you clarify?

Reply:
It actually doesn't make sense 😄 I forgot to fix it, good catch!